storybook/stories/API/shapes/Cross.stories.tsx

```diff
@@ -48,7 +48,7 @@ export default {
 
 export const Simple = {
   render: (args: Record<string, any>) => {
-    const CustomizedCross = (props: Record<string, any>) => <Cross {...props} />;
+    // const CustomizedCross = (props: Record<string, any>) => <Cross {...props} />;
```

Comment:
I think we can remove this since we're rendering the component on its own down below

```suggestion
```